Singleton/SceneInstance.cs

```diff
@@ -2,16 +2,16 @@
 
 public class SceneInstance<T> : MonoBehaviour where T : class, new() {
 	//
-	private static T _Instance = null;
+	protected static T _Instance = null;
```

Comment:
not hackish, it's inheritance XD
and i did need it in once instance, thus this change